### PR TITLE
Fix compile time warnings

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json [hunk 1]
```diff
@@ -15,22 +15,7 @@
...
-      "SANAPI": "free",
-      "SANBASE": "free"
-    },
-    "aggregation": "last",
-    "min_interval": "5m",
-    "table": "intraday_metrics",
-    "has_incomplete_data": true,
-    "data_type": "timeseries"
-  },
+
```
Comment (spiderjako) [inline lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json:18]: Unnecessary new line

File: lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json [hunk 2]
```diff
@@ -15,22 +15,7 @@
     "has_incomplete_data": true,
     "data_type": "timeseries"
   },
-  {
-    "human_readable_name": "ETH Price",
```
Comment (tspenov) [inline lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json:19]: Why do you remove this from json file ?

Comment (tspenov) [reply lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json:19]: Ok, I saw the reason

Comment (IvanIvanoff) [reply lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json:19]: It's duplicated with the proper metric in the price metric adapter, so when generating the functions in metric.ex it had warnings

File: test/sanbase/signals/validation_test.exs [hunk 3]
```diff
@@ -478,10 +478,8 @@ defmodule Sanbase.Signal.ValidationTest do
     alias Sanbase.Model.Project
 
     test "with binary slug, which is a project, returns :ok" do
-      project = Sanbase.Factory.insert(%Project{name: "santiment", slug: "santiment"})
-      project_slug = %{slug: "santiment"}
-
-      assert Validation.valid_slug?(project_slug) == :ok
+      project = Sanbase.Factory.insert(:random_project)
+      assert Validation.valid_slug?(project.slug) == :ok
```
Comment (tspenov) [inline test/sanbase/signals/validation_test.exs:482]:
```suggestion
      assert Validation.valid_slug?(%{slug: project.slug}) == :ok
```

**Pull request merged**